File: examADFD/src/main/java/com/example/examADFD/model/Place.java
package com.example.examADFD.model;

public class Place {
    private Long id;
    private String name;
    private String imageUrl;
    private String description;

    public Place(Long id, String name, String imageUrl, String description) {
        this.id = id;
        this.name = name;
        this.imageUrl = imageUrl;
        this.description = description;
    }

    public Long getId() { return id; }
    public String getName() { return name; }
    public String getImageUrl() { return imageUrl; }
    public String getDescription() { return description; }
}

File: front_end/lib/models/place.dart
class Place {
  final int id;
  final String name;
  final String imageUrl;
  final String description;

  Place({
    required this.id,
    required this.name,
    required this.imageUrl,
    required this.description,
  });

  factory Place.fromJson(Map<String, dynamic> json) {
    return Place(
      id: json["id"],
      name: json["name"],
      imageUrl: json["imageUrl"],
      description: json["description"],
    );
  }
}
